### PR TITLE
Sql homework

### PR DESCRIPTION
**Pull request by dmurasov**
The first 30 requests for the first task were completed and the files for the second task were added.

### CODE REVIEW

File: homework-selects/sql_questions.sql [hunk 1]
```diff
@@ -3,35 +3,236 @@
...
+
 -- 6. the first name of all employees including the first name of their manager.
+SELECT e.first_name AS Employee_first_name, m.first_name AS Manager_first_name
+FROM employees e
+    INNER JOIN employees m ON (m.employee_id = e.manager_id);
+
 -- 7. the first name of all employees and the first name of their manager including those who does not working under any manager.
+SELECT e1.first_name, e2.first_name AS manager_name 
+FROM employees e1 
+    LEFT JOIN employees e2 ON e1.employee_id = e2.manager_id;
```
Comment (perepiolcamaxim) [inline homework-selects/sql_questions.sql]: if e2 are managers and e1 employees, shouldn't it be e1.manager_id = e2.employee_id ?

File: homework-selects/sql_questions.sql [hunk 2]
```diff
@@ -3,35 +3,236 @@
...
 --10. the department name and number of employees in each of the department.
+SELECT department_name, COUNT(employee_id) AS number_of_employees 
+FROM departments 
+    INNER JOIN employees USING(department_id)
+GROUP BY department_name;
+
 --11. the name of the department, average salary and number of employees working in that department who got commission.
+SELECT department_name, AVG(salary) AS average_salary, COUNT(employee_id) AS number_of_employees 
+FROM departments 
+    LEFT JOIN (
```
Comment (perepiolcamaxim) [inline homework-selects/sql_questions.sql:68]: I believe left join will get departments even if commission_pct is null, I think we should use inner join

Comment (perepiolcamaxim) [reply homework-selects/sql_questions.sql:68]: and also why not simplify to this :
departments inner join employees using(department_id)
where commision_pct is not null

File: homework-selects/sql_questions.sql (hunk 1 above)
Comment (dmurasov) [reply homework-selects/sql_questions.sql]: was changed

File: homework-selects/sql_questions.sql (hunk 2 above)
Comment (dmurasov) [reply homework-selects/sql_questions.sql:68]: has not been changed

Comment (dmurasov) [reply homework-selects/sql_questions.sql:68]: because the query returns the correct information

**Review by perepiolcamaxim — APPROVED** [review]

**Pull request merged**